use-cases/org-existing-cloudtrail-ecs-vpc-subnet.md

```diff
@@ -76,17 +70,17 @@ module "utils_ecs-vpc" {
 ```
 -->
 
-1. Configure `AWS_PROFILE` with an organizational Administration credentials
+1. Configure `AWS_PROFILE` with an organizational administration credentials.
 
-2. Choose an Organizational **Member account for Sysdig Workload** to be deployed.
-   - This accountID will be provided in the `SYSDIG_SECURE_FOR_CLOUD_MEMBER_ACCOUNT_ID` parameter
-   - Use-case workload-related pre-existing resources (ecs,vpc,subnets) must live within this member account
+2. Choose an Organizational member account for Sysdig Workload to be deployed.
+   - Note the account ID of this account. This value will be provided in the `SYSDIG_SECURE_FOR_CLOUD_MEMBER_ACCOUNT_ID` parameter.
+   - Workload resources (ECS, VPC, subnets) must be created in this member account.
```

Comment:
its not that they must be created, since the use-case is about re-using existing resources.
relevant point to raise, is that they must be int he same member account